### PR TITLE
Add readme note about supporting streamers

### PR DESCRIPTION
**Pull request by gravyboat**
This is similar to what @bastimeyer has over on the gui: https://github.com/streamlink/streamlink-twitch-gui#why and I wanted to make sure we're noting that for the CLI as well. I've slightly modified what he had to make sure we're also linking Twitch's official stance regarding ad blockers.

### CODE REVIEW

**Comment by bastimeyer** [comment]
Wouldn't it be better to add a text that targets all plugins and not just Twitch? Despite that, I agree that something like this should be included in the readme here. Maybe even on the website?!

Btw, if you want to still link to the Twitch blog, you can use the short URL format which is a bit better to read:
https://blog.twitch.tv/-3ca5ce3287c

File: README.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ The main purpose of streamlink is to convert CPU-heavy flash plugins to a less C
 
 Streamlink is a fork of the [livestreamer](https://github.com/chrippa/livestreamer) project.
 
+Please note that by using this application you're bypassing any ads run by
+Twitch (adblock users do as well and
```
Comment (cdrage) [inline README.md]: Perhaps saying something along the lines of `...bypassing any ads ran by plugins such as Twitch`. So it still shows that we remain plugin-agnostic (no favouritism!) xD

**Comment by gravyboat** [comment]
Good suggestions, I'll update when I have time.

**Comment by gravyboat** [comment]
I've made some changes, let me know what you think @cdrage and @bastimeyer.

**Comment by cdrage** [comment]
LGTM! Although for some reason travis failed?

**Comment by cdrage** [comment]
Nvm; false alarm, going to restart the job

**Pull request merged**